### ISSUE TITLE
update to references list

### ISSUE THREAD

**Issue by FUF1** [leihs/leihs#1162]
*[Pädagogische Hochschule Zürich, University of Teacher Education Zurich (PHZH)](http://www.phzh.ch), Zürich, Switzerland* is using leihs as well.

**Comment by tmulzer** [comment]
Is "[Berliner Hochschule für Technik (BHT)](https://www.bht-berlin.de/), Berlin, Germany" on the list?

**Comment by FUF1** [comment]
> Is "[Berliner Hochschule für Technik (BHT)](https://www.bht-berlin.de/), Berlin, Germany" on the list? .. looks like we could join. ;)

I don't think it is. Only "Berliner Technische Kunsthochschule (btk), Berlin, Germany" is.
